### PR TITLE
switch to babel 6

### LINKED ISSUE #3: Does this work with babel6 ?
I am getting an error with babel6, the lodash version seems to be working great. I haven't diffed the changes between code base.

```
TypeError: Plugin is not a function
    at exports.default (/home/risto/my-app-name-here/node_modules/babel-plugin-ramda/lib/index.js:33:10)
    at Function.memoisePluginContainer (/home/lib/node_modules/babel-cli/node_modules/babel-core/lib/transformation/file/options/option-manager.js:1
27:13)
    at Function.normalisePlugin (/home/lib/node_modules/babel-cli/node_modules/babel-core/lib/transformation/file/options/option-manager.js:161:32)
    at /home/lib/node_modules/babel-cli/node_modules/babel-core/lib/transformation/file/options/option-manager.js:197:30
    at Array.map (native)
    at Function.normalisePlugins (/home/lib/node_modules/babel-cli/node_modules/babel-core/lib/transformation/file/options/option-manager.js:173:20)
    at OptionManager.mergeOptions (/home/lib/node_modules/babel-cli/node_modules/babel-core/lib/transformation/file/options/option-manager.js:271:36
)
    at OptionManager.mergeOptions (/home/lib/node_modules/babel-cli/node_modules/babel-core/lib/transformation/file/options/option-manager.js:303:10
)
    at OptionManager.addConfig (/home/lib/node_modules/babel-cli/node_modules/babel-core/lib/transformation/file/options/option-manager.js:221:10)
    at OptionManager.findConfigs (/home/lib/node_modules/babel-cli/node_modules/babel-core/lib/transformation/file/options/option-manager.js:364:16)

```

### PR DESCRIPTION
**Pull request by borisirota**
closes #3 

*I removed the push script from npm scripts because couldn't find something compatible in babel 6. Its possible to create a similar one from the [original babel-plugin publish command](https://github.com/babel/babel/blob/5.x/packages/babel-cli/src/babel-plugin/index.js#L135)*

### CODE REVIEW

**Comment by borisirota** [comment]
I've finished the work on this PR. It can be merged.

**Comment by megawac** [comment]
Thanks @borisirota I'll review tonight!

File: src/index.js [hunk 1]
```diff
@@ -29,34 +38,41 @@ export default function({ Plugin, types: t }) {
...
+        let { node, hub } = path;
+        let { name } = node.callee;
+        let { file } = hub;
         if (!t.isIdentifier(node.callee)) return;
-        if (specified[name]) {
-          node.callee = importMethod(specified[name], file);
-          return node;
+        if (specified[name]) node.callee = importMethod(specified[name], file);
+        if (node.arguments) {
+          node.arguments = node.arguments.map(arg => {
```
Comment (megawac) [inline src/index.js]: Nice! you cherry-picked https://github.com/lodash/babel-plugin-lodash/pull/11?

Comment (borisirota) [inline src/index.js]: yep :)

**Pull request merged**